static/js/board.js

```diff
@@ -1,41 +1,130 @@
 function popluateBoard(bobbles) {
-  var $bobbleList = $("#slideshow");
+  var $moments = $(".moments");
+  var $memes = $(".memes");
   for (var i = 0; i < bobbles.length; i++) {
-    $bobbleList.append(createBoardElement(bobbles[i]));
+    let bob = bobbles[i];
+    if (bob.flavor === "Moment") {
+      $moments.append(createBoardElement(bob));
+    } else if (bob.flavor === "Meme") {
+      $memes.append(createBoardElement(bob));
+    } else {
+      //temporary(undecided), where to attach bobs that aren't memes or moments?
+      $moments.append(createBoardElement(bob));
+    }
```

Comment:
Is it possible to merge the if statement with addBoardElement? The same code exists twice right after each other.